_assets/scripts/run_unit_tests.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+
+UNIT_TEST_PACKAGES=$(go list ./...  | \
+  grep -v /vendor | \
+  grep -v /t/e2e | \
+  grep -v /t/benchmarks | \
+  grep -v /transactions/fake )
+
+for package in ${UNIT_TEST_PACKAGES}; do
+  echo "Testing ${package}"
```

Comment:
Maybe some more color here too?
```suggestion
  echo -e "${GRN}Testing:${RST} ${package}"
```